SourceS/miniwin/com/dsound_macro.inc

```diff
@@ -25,4 +25,4 @@
 #define DSERR_CONTROLUNAVAIL MAKE_DSHRESULT(30)
 
 #define DSSCL_EXCLUSIVE DVL_DSSCL_EXCLUSIVE
-#define DVL_DSBSTATUS_PLAYING DSBSTATUS_PLAYING
+#define DVL_DSBSTATUS_PLAYING 0x00000001
```

Comment:
Nice, with this we can remove an `#ifdef` from the Switch and the RetroFW ports.

Reply:
This was also an issue for Nintendo Switch. It's the little things some times :)